frontend/src/utils/saveLocalStorageData.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+import { DegreeSliceState } from 'reducers/degreeSlice';
+import { PlannerSliceState } from 'reducers/plannerSlice';
+import prepareLocalStorageData from './prepareLocalStorageData';
+
+const saveLocalStorageData = (): void => {
```

Comment:
nitpict: can we rename this util function to `migrateLocalStorageData`? Might be a better name imo since `saveLocalStorageData` sounds like its saving data rather than migrating it?